schemas/dependencies/dynatrace-environment-1.yml

```diff
@@ -14,9 +14,27 @@ properties:
   description:
     type: string
   environmentUrl:
+    description: the URL to a specific Dynatrace entity called environment, which is also known as tenant.
     type: string
   bootstrapToken:
     "$ref": "/common-1.json#/definitions/vaultSecret"
+  environment:
+    type: string
+    enum:
+    - integration
+    - stage
+    - production
+  dynatraceRegion:
+    type: string
+    enum:
+    - us-east-virginia
+    - eu-west-ireland
+    - ap-southeast-singapore
+  awsRegion:
+    description: a list of HCP AWS regions
+    type: array
+    items:
+      type: string
```

Comment:
Should these new fields map to a gql type too?

Reply:
You mean `schemas/aws/regions-1.yml`?

Reply:
My understanding is we will need these as gql types too, e.g., [here](https://github.com/app-sre/qontract-schemas/blob/fda16f3a50edb8fe1e4345cbe0a8ec15f66a1fc0/graphql-schemas/schema.yml#L586)?

Reply:
Ah yes.